### ISSUE TITLE
How do I initialize the ol.map using my own class myMap (using inheritance), in OpenLayers 3?

### ISSUE THREAD

**Issue by joaorodr84** [openlayers/openlayers#2094]
I am working on a GIS API and I am changing from OpenLayers 2 to Openlayers 3.

I want to instantiate the ol.map using my own class, like this:

**myMap.js**

```
goog.inherits(myMap, ol.map);
```

**index.html**

```
new myMap({
    layers: [{ ... }]
    ...
})
```

Unfortunately, the map doesn't show up in the browser. Nothing happens.

Do I have to execute some function in the myMap init, like in OpenLayers 2? Something like "this.prototype.parent.initialize ..."??

This is how ol.map instantiation normally works:
[http://jsfiddle.net/joaorodr84/SrBWL/](http://jsfiddle.net/joaorodr84/SrBWL/)

This is how I want it to works, using my own "myMap" class:
[http://jsfiddle.net/joaorodr84/T2Bet/11/](http://jsfiddle.net/joaorodr84/T2Bet/11/)

Basically, I want to instantiate the map using my own class "myMap", like we did it in OpenLayers 2 with "OpenLayers.Class".

I know this isn't a real issue, but I have asked in the ol3 google group and I've got no answer.

Could you please help me with this?

Thank you.

João
